sympy/functions/special/spherical_harmonics.py

```diff
@@ -90,8 +91,8 @@ class Ynm(Function):
 
     >>> from sympy import Ynm, Symbol, diff
     >>> from sympy.abc import n,m
```

Comment:
This part can be removed afaik. I think it is only required to import diff here. (Or one can simply write `Ynm(n, m, theta, phi).diff(theta)`)